### PR TITLE
19724_2 - EFT - Create CFS account job 

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/19724

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: pay-api/src/pay_api/services/eft_service.py [hunk 1]
```diff
@@ -48,6 +48,15 @@ def create_account(self, identifier: str, contact_info: Dict[str, Any], payment_
         cfs_account.status = CfsAccountStatus.PENDING.value
         return cfs_account
 
+    def create_account(self, identifier: str, contact_info: Dict[str, Any], payment_info: Dict[str, Any],
+                       **kwargs) -> CfsAccountModel:
+        """Create an account for the EFT transactions."""
+        # Create CFS Account model instance, set the status as PENDING
```
Comment (seeker25) [inline pay-api/src/pay_api/services/eft_service.py:54]: redundant

File: jobs/payment-jobs/tasks/cfs_create_account_task.py [hunk 2]
```diff
@@ -101,8 +101,12 @@ def _create_cfs_account(cls, pending_account: CfsAccountModel, pay_account: Paym
                 'bankAccountNumber': pending_account.bank_account_number,
                 'bankAccountName': pay_account.name
             }
-
-            if pending_account.cfs_account and pending_account.cfs_party and pending_account.cfs_site:
+            # switch from PAD to EFT
```
Comment (seeker25) [inline jobs/payment-jobs/tasks/cfs_create_account_task.py:104]: Not sure what this comment means? Is the branch below only when you switch from PAD to EFT?

What happens if they create a new account that's EFT without being PAD first?

**Review by seeker25 — APPROVED** [review]

**Comment by seeker25** [comment]
CI is looking at the old branch :(

File: pay-api/src/pay_api/services/eft_service.py (hunk 1 above)
Comment (Jxio) [reply pay-api/src/pay_api/services/eft_service.py]: removed

File: jobs/payment-jobs/tasks/cfs_create_account_task.py (hunk 2 above)
Comment (Jxio) [reply jobs/payment-jobs/tasks/cfs_create_account_task.py]: removed

**Pull request merged**